PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,8 @@
 ## What does this change?
+<!-- Ideally, a PR will have enough detail to continue to be understandable by future travellers. Think about describing why the change is needed, how the change addresses the problem and any questions or points of discussion. -->
```

Comment:
```suggestion
<!-- A PR should have enough detail to be understandable far in the future. e.g what is the problem/why is the change needed, how does it solve it and any questions or points of discussion. -->
```

Reply:
just trying to remove words without changing the meaning.  I was considering also suggesting self-review of the actual code, to pull out any significant areas for extra focus.

Reply:
Much clearer, thanks!

> I was considering also suggesting self-review of the actual code, to pull out any significant areas for extra focus.

This is super useful, I was thinking `any questions or points of discussion` would act as a prompt for this?